samples/bluetooth/central_hids/README.rst

```diff
@@ -115,7 +116,7 @@ Testing with nRF Connect for Desktop
 #. Go to the **Connection Map** tab.
    Click the dongle configuration and select **Advertising setup**.
 
-   The current version of nRF Connect cannot store the advertising setup, so we must configure it manually.
+   The current version of nRF Connect cannot store the advertising setup, so it must be configured manually.
    See the following image for the required target configuration:
 
    .. figure:: /images/bt_central_hids_nrfc_ad.png
```

Comment:
This way of including the image works just fine at the moment, but seems to fail when running with the zephyr master. So it will probably fail after a rebase.
@carlescufi How should we include images? See Jira NCSDK-1047.

Reply:
Fixed 'we must configure it' -> 'it must be configured'
BTW: wouldn't it sounds better to use "has to": 'it has to be configured'?

Reply:
No, we're using simple English. "Must" is much clearer than "have to".